python/tk_desktop/command_panel/command_button.py

```diff
@@ -162,7 +162,7 @@ def add_command(self, command_name, menu_name, icon, tooltip, is_menu_default):
         # If there is more than one available item in the menu, show it so the
         # user can pick one.
         if len(self._menu.actions()) > 1:
-            self.setPopupMode(self.MenuButtonPopup)
+            self.setPopupMode(QtGui.QToolButton.MenuButtonPopup)
```

Comment:
This is the only change related to PySide6. We're using the enumerator class instead of using it as object attribute. This is backwards compatible as CI confirmed.